docs/source/contributing/ci.adoc

```diff
@@ -41,7 +41,7 @@ On top of building the master branch and pull requests, CentOS CI is also used t
 
 
 |link:https://ci.centos.org/job/minishift/[master], link:https://ci.centos.org/job/minishift-pr/[pull requests], link:https://ci.centos.org/job/minishift-docs/[docs],
-link:https://ci.centos.org/job/minishift-release/[release]
+link:https://ci.centos.org/job/minishift-release/[release], link:https://ci.centos.org/job/minishift-nightly[nightly]
```

Comment:
👍

Reply:
So what happened to the extra explanation? There was useful content in the previous version of this change. It's all gone now.

Reply:
You did not update the description column